src/Orc.NuGetExplorer.Xaml/ViewModels/PackageSourceSettingViewModel.cs

```diff
@@ -189,6 +192,10 @@ private void SaveFeeds()
             SettingsFeeds.AddRange(Feeds);
 
             _configurationService.SavePackageSources(Feeds);
+            foreach(var feed in RemovedFeeds)
```

Comment:
Make sure that this collection cannot be modified to prevent "collection has changed" exceptions.